### DISCUSSION TITLE
TX2 gpu memory cost abnormal

### DISCUSSION THREAD

**Discussion by beingjoey** [apache/incubator-mxnet#17132] [Q&A]
## Description
I built mxnet in Tx2 , and did mtcnn face detecting and insightface , and I found the memory cost in TX2 was about 2 times of it in x86 computers which equiped nvidia-t2080 or nvidia 1050,(on these 2 computers the memory cost were almost the same).
My test used the same image and mtcnn face detecting costed about 1GB memory, compared with 500MB in computer.

**Comment by samskalicky** [comment]
@mxnet-label-bot update [Question]
Whats the expectation in MXnet for memory usage on arm vs x86?

**Comment by beingjoey** [comment]
> @mxnet-label-bot update [Question]
> Whats the expectation in MXnet for memory usage on arm vs x86?

the same or even lower would be better.

**Comment by litaotju** [comment]
Which memory are you referencong here? TX2 shares gpu and host cpu memory. While in x86 the cpu and gpu does not share mem
